src/pca.js

```diff
@@ -110,6 +110,22 @@ export class PCA {
     return predictions.subMatrix(0, predictions.rows - 1, 0, nComponents - 1);
   }
 
+  /**
+   * Calculates the inverse PCA transform
+   * @param {Matrix} dataset
+   * @param {Array} mean
+   * @return {Matrix} dataset projected in the PCA space
+   */
+  invert(dataset, mean) {
+    dataset = new Matrix(dataset);
```

Comment:
`Matrix.checkMatrix(dataset)`

This will avoid copying the data if `dataset` is already an instance of `Matrix`